x-pack/plugins/observability_ai_assistant/public/components/chat/welcome_message_connectors.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { css, keyframes } from '@emotion/css';
+import { EuiBetaBadge, EuiButton, EuiSpacer, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { euiThemeVars } from '@kbn/ui-theme';
+import type { UseGenAIConnectorsResult } from '../../hooks/use_genai_connectors';
+
+const fadeInAnimation = keyframes`
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+`;
+
+const fadeInClassName = css`
+  animation: ${fadeInAnimation} ${euiThemeVars.euiAnimSpeedNormal} ease-in-out;
+`;
+
+export function WelcomeMessageConnectors({
+  connectors,
+  onSetupConnectorClick,
+}: {
+  connectors: UseGenAIConnectorsResult;
+  onSetupConnectorClick?: () => void;
+}) {
+  return !connectors.loading && connectors.connectors?.length === 0 && onSetupConnectorClick ? (
```

Comment:
Could we flip away this ternary and use a guard instead?

So check for the null case first and return early, else return the markup, it just makes the indent easier to read and makes it clear that we do nothing if the condition is false